contracts/build.gradle

```diff
@@ -71,10 +71,15 @@ cordapp {
     contract {
         name "ContractsModuleNameHere"
         versionId 1
-        vendor "VendorNameHere"
+        licence cordappLicense
+        vendor cordappVendorName
     }
 }
 
+// Use the name of the CorDapp as the name of the JAR file.
```

Comment:
You're referring to JAR files a few times, but is that correct?
Are we not in CPK/CPB/CPI territory?

Reply:
well if it's a cpb, it's not a jar right?

Reply:
i think I'm using cpk and cpb incorrectly. *runs to read docs*

Reply:
Also some examples have a space in the `name` field which isn't allowed for generating jar files. Suggestions on names for the contracts module? I'll add a `contractsModule` variable with `contracts` as the default

https://github.com/corda/samples-kotlin/blob/8ce33f57ad1dd77f721af4d1f59d72f3d331fd0c/Tokens/tokentofriend/contracts/build.gradle#L7

Reply:
@tlawson3 just updated. Pls review

Reply:
<img width="1264" alt="Screenshot 2023-07-20 at 15 07 49" src="https://github.com/corda/CSDE-cordapp-template-kotlin/assets/18750678/d26470fe-ee7b-4274-8059-ceb3ac5552cc">